packages/fxa-auth-server/lib/senders/templates/post_verify_trailhead.txt

```diff
@@ -1,6 +1,6 @@
 {{t "Welcome to the Firefox family" }}
 
-{{t "You're signed in and ready to start exploring safely and securely." }}
+{{{t "You're signed in and ready to start exploring safely and securely." }}}
```

Comment:
Bonus fix, the `'` was being encoded as `&#x27;` even though this is supposed to be a plaintext email.